nozzle/config.go

```diff
@@ -0,0 +1,134 @@
+package nozzle
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/cloudfoundry/sonde-go/events"
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Config holds users provided env variables
+type Config struct {
+	Nozzel    *NozzelConfig
+	WaveFront *WaveFrontConfig
+}
+
+// NozzelConfig holds specific PCF env variables
+type NozzelConfig struct {
+	APIURL                 string `required:"true" envconfig:"api_url"`
+	Username               string `required:"true"`
+	Password               string `required:"true"`
+	FirehoseSubscriptionID string `required:"true" envconfig:"firehose_subscription_id"`
+	SkipSSL                bool   `default:"false" envconfig:"skip_ssl"`
+
+	SelectedEvents []events.Envelope_EventType `ignored:"true"`
+}
+
+// WaveFrontConfig holds specific Wavefront env variables
+type WaveFrontConfig struct {
+	URL           string `envconfig:"URL"`
```

Comment:
Okay for testing. But we'd want the nozzle to only go through the proxy. We can probably remove this before we merge.

Reply:
We should leave it, if the users use the tile those options will not show on the form, so they can't use... and for dev and testing is very useful to reduce deploy time.